mlrun/feature_store/steps.py

```diff
@@ -66,8 +98,32 @@ def do(self, event):
                     )
         return event
 
+    def _do_pandas(self, event):
+        body = event.body
+        for column in body:
+            for name, validator in self._validators.items():
+                if name == column:
+                    violations = 0
+                    all_args = []
+                    for i in range(body[column].shape[0]):
+                        # check each body entrance if there is validator for it
```

Comment:
```suggestion
                        # check each body entry if there is validator for it
```
if I understand it correctly.